Automattic/wp-calypso
# Domains management i1: Bulk auto renew

Related to https://github.com/Automattic/dotcom-forge/issues/3428


![CleanShot 2023-08-23 at 15 09 24@2x](https://github.com/Automattic/wp-calypso/assets/1500769/134b7a86-268e-40c1-83d6-25f6793c0a28)

![CleanShot 2023-08-23 at 15 09 48@2x](https://github.com/Automattic/wp-calypso/assets/1500769/ab7c952d-65d7-4c35-b1fc-89ac5aa65734)



## Proposed Changes

* A bulk actions toolbar appears whenever a domain is selected
* Bulk auto-renew actions appear in a dropdown
* Makes network call to begin bulk action (you can view the request in the network tools, backend may not be complete)
* Adds a spacer to the top of the table where the search/filtering toolbars will eventually go

This PR doesn't add any progress, but that's coming as part of https://github.com/Automattic/dotcom-forge/issues/3419

The dropdown width changes a little bit when you go from having one domain selected to two domains selected. It happens because because text inside the dropdown changes from "... 1 domain" to "... 2 domains". I can't think of an elegant fix for this that works with translations.

## Testing Instructions



* Apply D119397-code and follow instructions to sandbox the async jobs
* Test the behaviour of selecting domains
* Feel free to click the button, although the underlying setting won't change
* Design should be pretty close to 31819-pb

## Pre-merge Checklist
